playbook/app/pb_kits/playbook/pb_icon/_icon.tsx

```diff
@@ -41,19 +40,6 @@ type IconProps = {
   spin?: boolean,
 } & GlobalProps
 
-type AliasType = string | string[];
```

Comment:
I'm a bit confused? Why did we remove all of the logic from the icon alias work. It still needs to live inside the playbook icon kit, even if the actual alias list lives in the `playbook-icons` repo.

Reply:
I see now that a lot was moved to the playbook-icons library. I'll check that out now.